src/executables/csql.c

```diff
@@ -1484,7 +1484,7 @@ csql_do_session_cmd (char *line_read, CSQL_ARGUMENT * csql_arg)
 	}
       else
 	{
-	  fprintf (csql_Output_fp, "CONNECT session command cannot support --sysadm mode\n");
+	  fprintf (csql_Output_fp, "CONNECT session command cannot support --SA-mode and --sysadm mode\n");
```

Comment:
CONNECT session command **does not support** --SA-mode and --sysadm mode

I propose a classification of error messages, for example:
```
          fprintf (csql_Output_fp, "CONNECT session command does not support %s %s.\n",
                csql_arg->sa_mode ? "--SA-mode" : "", csql_arg->sysadm ? "(--sysadm mode)" : "");
 
```

Reply:
It is better to apply each case.(sa and sysadm)
I will apply it thank you.